assignments/assignment03.rb

```diff
@@ -7,6 +7,22 @@
 
 # re-implement titleize and palindrome? as methods on String
 
+class String
+  def titleize(s)
+    words = s.split
+    caps = []
+    words.each do |word|
+      caps << word.capitalize
+      end
+    caps.join " "
+  end
+
+  def palindrome?(s)
```

Comment:
palindrome? won't accept a parameter, since it's operating on the instance of the string itself